tests/reactive_fluid_transport_zero_solubility_porosity/screen-output

```diff
@@ -0,0 +1,60 @@
+[Daniels-MacBook-Pro-3.local:50962] shmem: mmap: an error occurred while determining whether or not /var/folders/zw/0t9y_fkn22l4df48w2pgzd280000gn/T//ompi.Daniels-MacBook-Pro-3.501/jf.0/509083648/sm_segment.Daniels-MacBook-Pro-3.501.1e580000.0 could be created.
+-----------------------------------------------------------------------------
+-----------------------------------------------------------------------------
+-----------------------------------------------------------------------------
+
+-----------------------------------------------------------------------------
+-----------------------------------------------------------------------------
+Number of active cells: 1 (on 1 levels)
+Number of degrees of freedom: 74 (18+7+18+4+9+9+9)
+
+*** Timestep 0:  t=0 years, dt=0 years
+   Solving temperature system... 0 iterations.
+   Solving porosity system ... 0 iterations.
+   Solving bound_fluid system ... 0 iterations.
+   Rebuilding Stokes preconditioner...
+   Solving Stokes system (AMG)... 2+0 iterations.
+   Solving fluid velocity system... 1 iterations.
+      Relative nonlinear residuals (temperature, compositional fields, Stokes system): 8.76385e-17, 1.96911e-17, 1.96911e-17, 2.59112e-05
+      Relative nonlinear residual (total system) after nonlinear iteration 1: 2.59112e-05
+
+
+   WARNING: The nonlinear solver in the current timestep failed to converge.
+   Acting according to the parameter 'Nonlinear solver failure strategy':
+   Continuing to the next timestep even though solution is not fully converged.
+
+   Postprocessing:
+     Melt fraction min/total/max: 0.01669, 6.676e+04, 0.01669
```

Comment:
I noticed that after the reaction (in timestep 1) the number is exactly what you say above, whereas here (before the reaction), it is a little bit higher. Any idea why that might be the case? Since the equilibrium melt fraction for zero solubility only depends on the bulk water content, I would have though the numbers should be exactly the same.